src/view/gui/fDCM2NIfTI.cpp

```diff
@@ -30,7 +30,7 @@ fDCM2NIfTIConverter::fDCM2NIfTIConverter()
   }
 
 #elif __linux__
-  m_exe =  "." + QApplication::applicationDirPath() + "/dcm2niix";
+  m_exe =  /*"." + */QApplication::applicationDirPath() + "/dcm2niix";
```

Comment:
Actually, this will not work for macOS (yay for the fruit company). You need to use `getApplicationPath("dcm2niix")` to get it working across platforms.

Reply:
This works on Windows and Linux from GUI. The dcm2nii call we had till now is without using getApplicationPath("dcm2nii") and all the path magic was happening in this file. Was dcm2nii call really working on macOS earlier?  If it was working earlier, it should work even now. We will need to test on macOS and I am not sure how to test the GUI there.

Reply:
> This works on Windows and Linux from GUI

I am not disputing this. What I said was that it won't work on macOS. This is because the binaries are not in `QApplication::applicationDirPath()` for the macOS package (it is in `../Resources/bin/`). All other "packaged" external binaries are called using `getApplicationPath` to take care of this.

Reply:
As long as this is taken care of, these changes are great.

Reply:
> 
> 
> > This works on Windows and Linux from GUI
> 
> I am not disputing this. What I said was that it won't work on macOS (BTW, the mechanism on the current master wouldn't work, either). This is because the binaries are not in `QApplication::applicationDirPath()` for the macOS package (it is in `../Resources/bin/`). All other "packaged" external binaries are called using `getApplicationPath` to take care of this.

The existing code may have never worked on macOS then.

Reply:
@sarthakpati please check the modified file.